pkg/batcher/batcher.go

```diff
@@ -138,6 +147,7 @@ func (batcher *AsynchronousBatcher) run() {
 
 // SubmitComponent submits a component to the batch
 func (batcher AsynchronousBatcher) SubmitComponent(checkID check.ID, instance topology.Instance, component topology.Component) {
+	log.Tracef("Submitting component for check [%s] instance [%s]: %s", checkID, instance.GoString(), component.JSONString())
```

Comment:
will this marshall each and every component? (i think it will, because arguments are strict in go). We should use log.shouldLog(seelog.TraceLvl)with a conditional to avoid this

Reply:
we will drop it